projects/plugins/protect/src/js/routes/scan/scan-admin-section-hero.tsx

```diff
@@ -119,7 +119,7 @@ const ScanAdminSectionHero: React.FC = () => {
 				</Text>
 				<OnboardingPopover
 					id={ hasPlan ? 'paid-daily-and-manual-scans' : 'free-daily-scans' }
```

Comment:
At one point, this paid popover pointed at the `Scan Now` button - seems clear enough where it is currently highlighting the `lastCheckedLocalTimestamp`, but is this something we maybe want to re-establish, if possible?